libs/libxx/Kconfig

```diff
@@ -22,6 +22,34 @@ config HAVE_CXX
 
 if HAVE_CXX
 
+choice
+	prompt "C++ Library"
+	default NUTTXCXX
+
+config NUTTXCXX
+	bool "NuttX C++ library"
```

Comment:
i just named this version by myself. if you have a better name suggestion, please.

Reply:
maybe NUTTX_LIBXX?

Reply:
how about NUTTX_LIBCXX to make it look a bit aligned to other twos?

Reply:
I think that it is better to stick to the naming already in place (libxx -> NuttX, libcxx -> LLVM). I don't like it, but it is better to avoid confusions.

Reply:
How about MINICXX or NXLIBCXX? nx is used in many place. But, anyway since NuttX's version is default, the name isn't really used.

Reply:
Please just be consistent. Otherwise when looking at the macro name you will not know what it refers to. A rename should involve renaming many other things.

Reply:
i changed it to NUTTX_LIBXX as it seems the least controversial for now.

Reply:
@v01d are you OK with the new name?

Reply:
shouldn't this require a rename in Makefiles depending on these variables? otherwise you can just have the change into a `choice` in this PR and have a separate PR make all necessary renames

Reply:
@v01d there isn't rename at all, NUTTXCXX is a new created name which equals to UCLIBCXX=n and LIBCXX==n in the old Kconfig.

Reply:
Ah, got it! Will merge then.